projects/extension/src/background/index.ts

```diff
@@ -12,17 +12,55 @@ import {
   ToExtension,
 } from "@substrate/connect-extension-protocol"
 
+const polkadot_cp = Object.assign({}, polkadot)
+const ksmcc3_cp = Object.assign({}, ksmcc3)
+const westend2_cp = Object.assign({}, westend2)
+const rococo_cp = Object.assign({}, rococo_v2_2)
+
+const setupStorageBootnodes = () => {
```

Comment:
I do not understand what the code suggestion above means

Reply:
You don't need the `export const wellKnownChains: Map<string, string>` anymore.
The only place where `wellKnownChains` is used is in `whenInitFinished` below, so instead of using `whenInitFinished` in `whenInitFinished` you can just call this function and use what it returns.
This also makes it possible to make this function async.